proto/engine/v1/execution_engine.proto

```diff
@@ -142,3 +142,20 @@ message Withdrawal {
 	uint64 amount = 4;
 }
 
+// Blobs bundle is retrieved through engine-api from the execution layer client.
+// It consists of the necessary components for constructing a blobs sidecar object to gossip through p2p.
+message BlobsBundle {
+	// The block hash of the payload  which corresponds to the blobs.
+	bytes block_hash = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+	// The KZG commitments of the blobs.
+	repeated bytes kzgs = 2 [(ethereum.eth.ext.ssz_size) = "?,48", (ethereum.eth.ext.ssz_max) = "16"];
```

Comment:
Does this have to be named kzgs? How about calling it commitments? `kzgs` is hard type, say, and it's not very idiomatic.

Reply:
Yeah. I also like commitments better

Reply:
Discussed offline. `kzg_commitments` is 👑